registry/scheduledTasks.xml

```diff
@@ -43,4 +43,8 @@
 		<descr>Automatically deposit any outstanding DOIs to the configured registration agency.</descr>
 		<frequency hour="0" />
 	</task>
+  <task class="lib.pkp.classes.task.RemoveUnvalidatedExpiredUsers">
```

Comment:
Sorry to bug you on trivial details but this file uses tabs for indentation and we should keep consistent :)

Reply:
fixed .